src/main/resources/swagger/api-docs.yaml

```diff
@@ -1974,10 +1974,17 @@ paths:
       summary: |
         deletes and rebuilds the entire index of published library datasets
       produces:
-      - text/plain
+        - text/plain
+        - application/json
       responses:
         200:
-          description: Success
+          description: List of errors if partial success, "()" if all datasets indexed successfully
```

Comment:
It sounds a little strange to get a 200 with some errors, but I'm guessing that's what rawls is giving us back so w/e.